src/zenml/utils/pipeline_docker_image_builder.py

```diff
@@ -80,16 +80,11 @@ def _include_global_config(
         fileio.rmtree(config_path)
 
 
-class PipelineDockerImageBuilder(BaseModel):
-    """Builds Docker images to run a ZenML pipeline.
+class PipelineDockerImageBuilderConfigMixin(BaseModel):
```

Comment:
I suggest leaving the `PipelineDockerImageBuilder` class untouched and without a separate config class. The only reason that stack components subclass this is the `docker_parent_image` attribute which was kept for backwards compatability. In this breaking release instead I suggest removing this attribute entirely and have users use the docker configuration instead.

Reply:
Makes sense. How should we fix this for now? I assume the orchestrators/step operators still need to subclass this class, so we simply remove the `docker_parent_image` attribute and all logic associated to it?